src/agent/src/test_utils.rs

```diff
@@ -0,0 +1,54 @@
+#[cfg(test)]
```

Comment:
Please can you add a copyright header to this new file.

Reply:
Sure, I'm adding the header.